Git-commit: ea30c1deb6c6ccbceb29b6a3d48e1f601aefddf1
Patch-mainline: v4.1-rc1
Reference: fate#317990
From: Brian King <brking@linux.vnet.ibm.com>
Date: Thu, 26 Mar 2015 11:23:56 -0500
Subject: [PATCH 28816/36546] ipr: Re-enable write same

Re-enable write same support for ipr RAID adapters.

Signed-off-by: Brian King <brking@linux.vnet.ibm.com>
Reviewed-by: Wen Xiong <wenxiong@linux.vnet.ibm.com>
Reviewed-by: Daniel Kreling <kreling@linux.vnet.ibm.com>
Signed-off-by: James Bottomley <JBottomley@Odin.com>
Acked-by: Dinar Valeev <dvaleev@suse.com>
---
 drivers/scsi/ipr.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/scsi/ipr.c
+++ b/drivers/scsi/ipr.c
@@ -6476,7 +6476,6 @@ static struct scsi_host_template driver_
 	.shost_attrs = ipr_ioa_attrs,
 	.sdev_attrs = ipr_dev_attrs,
 	.proc_name = IPR_NAME,
-	.no_write_same = 1,
 };
 
 /**
